sdccc/src/test/java/com/draeger/medical/sdccc/messages/TestMessageStorage.java

```diff
@@ -1820,10 +1819,74 @@ void testDetermineCharsetFromMessageFromHttpHeader(@TempDir final File dir) thro
         }
     }
 
+    @Test
+    void testDetermineCharsetFromMessageFromHttpHeaderSuppressEncodingErrors(@TempDir final File dir)
```

Comment:
Change the access modifier to _public_ and add javadoc.

Reply:
Done.